include/seqan3/core/detail/strong_type.hpp

```diff
@@ -390,6 +391,33 @@ class strong_type
     }
     //!\}
 
+    /*!\name Comparison operators.
```

Comment:
```suggestion
    /*!\name Comparison operators
```
No dots at the end of name sections.